src/main/java/homework3/Task1.java

```diff
@@ -0,0 +1,18 @@
+package homework3;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int a = 13;
+        float b = 6.55f;
+        float SumResult = a+b;
+        float SubtractionResult = a-b;
```

Comment:
Keep spaces between operator and operands
```suggestion
        float subtractionResult = a - b;
```
Or use "Format Code" Intelij IDE feature